stdlib/3/ast.pyi

```diff
@@ -19,11 +19,128 @@ else:
 class NodeVisitor:
     def visit(self, node: AST) -> Any: ...
     def generic_visit(self, node: AST) -> Any: ...
-    if sys.version_info >= (3, 8):
+    def visit_Module(self, node: Module) -> Any: ...
+    def visit_Interactive(self, node: Interactive) -> Any: ...
+    def visit_Expression(self, node: Expression) -> Any: ...
+    def visit_Suite(self, node: Suite) -> Any: ...
```

Comment:
There is an open issue about this in the bugtracker, https://bugs.python.org/issue39639

Reply:
I did wonder what this one was. If it's removed from python, will need to be made conditional I suppose.